doc/releases/quincy.rst

```diff
@@ -5,6 +5,537 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+
+v17.2.6 Quincy
+==============
+
+This is the sixth backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+* `ceph mgr dump` command now outputs `last_failure_osd_epoch` and
+  `active_clients` fields at the top level.  Previously, these fields were
+  output under `always_on_modules` field.
+
+Changelog
+---------
+
+* [bluestore] Improve deferred write decision (`pr#49333 <https://github.com/ceph/ceph/pull/49333>`_, Adam Kupczyk, Igor Fedotov)
+* [quincy] rgw/cloud-transition: Fix issues with MCG endpoint (`pr#49061 <https://github.com/ceph/ceph/pull/49061>`_, Soumya Koduri)
+* Add per OSD crush_device_class definition (`pr#50444 <https://github.com/ceph/ceph/pull/50444>`_, Francesco Pantano)
+* ceph-crash: drop privleges to run as "ceph" user, rather than root (CVE-2022-3650) (`pr#48805 <https://github.com/ceph/ceph/pull/48805>`_, Tim Serong, Guillaume Abrioux)
```

Comment:
"privileges" misspelled, missing an "i".

Reply:
Good catch, @icepic.

Reply:
Don't we have a precedent of leaving the PR titles as-is so they correlate with git history?

Mind you I assert that these things should be caught during PR approval.